src/components/TasksList/TasksList.tsx

```diff
@@ -1,9 +1,35 @@
 import React from 'react';
+import AppBar from '@material-ui/core/AppBar';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import TabPanel from './TabPanel';
 
 const TasksList: React.FC = () => {
+  const [value, setValue] = React.useState(0);
+
+  const handleChange = (event: React.ChangeEvent<{}>, newValue: number) => {
+    setValue(newValue);
+  };
+
   return (
     <div data-testid='task-list-page'>
-      <h1>Tasks List</h1>
+      <AppBar component='section' position='static' color='default'>
+        <Tabs
+          value={value}
+          onChange={handleChange}
+          indicatorColor='primary'
+          textColor='primary'
+          variant='fullWidth'>
+          <Tab label='Newbie Tasks' />
+          <Tab label='My tasks' />
+        </Tabs>
+      </AppBar>
+      <TabPanel value={value} index={0}>
+        Newbie Tasks Component
+      </TabPanel>
+      <TabPanel value={value} index={1}>
+        My Tasks Component
+      </TabPanel>
```

Comment:
And the same is here: `{Tabs.map(({ component }, index) => ( <TabPanel value={value} index={index} key={index}>{component}</TabPanel> ))}`

Thus it will be more generic and in the future we can easily add a new object to Tabs const, for example Talent Tasks Component